QuadPaySDK/Factories.swift

```diff
@@ -18,6 +18,34 @@ extension UIFont {
     func bold() -> UIFont {
         return withTraits(traits: .traitBold)
     }
+
+    public static func jbs_registerFont(withFilenameString filenameString: String, bundle: Bundle) {
```

Comment:
Why is this named `jbs_`?  What does `jbs` stand for?

Reply:
Rename the function to registerFont